superset/migrations/versions/7467e77870e4_remove_aggs.py

```diff
@@ -0,0 +1,44 @@
+"""remove_aggs
+
+Revision ID: 7467e77870e4
+Revises: c829ff0b37d0
+Create Date: 2018-07-22 08:50:01.078218
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '7467e77870e4'
+down_revision = 'c829ff0b37d0'
+
+
+def upgrade():
+    with op.batch_alter_table('table_columns') as batch_op:
+        batch_op.drop_column('avg')
+        batch_op.drop_column('max')
+        batch_op.drop_column('sum')
+        batch_op.drop_column('count_distinct')
+        batch_op.drop_column('min')
+
+    with op.batch_alter_table('columns') as batch_op:
+        batch_op.drop_column('avg')
+        batch_op.drop_column('max')
+        batch_op.drop_column('sum')
+        batch_op.drop_column('count_distinct')
+        batch_op.drop_column('min')
+
+
+def downgrade():
```

Comment:
I'm wondering if we should to a better work in `downgrade` for scripts that drop columns, because if I run an `upgrade` and then rollback with a `downgrade` I will have lost data. What if the upgrade function made a copy of the table before dropping columns, and the downgrade function recovered the data after recreating the columns? Is that a common pattern with alembic, or people just worry about schemas?